### ISSUE TITLE
Moderation: Need for more explanation for those who've had a contribution reported? 

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#463]
Currently this is all we say to them: 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/2b93ced2-efe6-4433-afbe-e07a4c6c6c63)

Whereas for people who've had a topic reported, we explain all this: 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/b7a9337e-6eaa-42b3-9174-9297ce552076)

Might need some consideration?

**Comment by ilmartyrk** [comment]
@anettlinno what do you think about that?

**Comment by anettlinno** [comment]
IMO for arguments the explanation "You can leave it as it is...." is enough. 
@BeccaMelhuish what do you think is missing there? Step-by-step explanation?

**Comment by BeccaMelhuish** [comment]
@anettlinno Shall we add Phase II label so it can be 'parked' for now but considered if we do a review of the moderation system (much-needed, I found when reviewing the texts) in Phase II or whenever the time comes? (Don't have my head in the moderation flow at the moment, and it's not urgent I think) :)

**Comment by anettlinno** [comment]
Yes, let´s do it like that. Sending this issue to Phase II for further discussion.
